seed/challenges/02-data-visualization-certification/data-visualization-projects.json

```diff
@@ -119,18 +119,16 @@
     },
     {
       "id": "bd7198d8c242eddfaeb5bd13",
-      "title": "Show Relationships with a Force Directed Graph",
+      "title": "Show National Contiguity with a Force Directed Graph",
       "description": [
-        "<strong>Objective:</strong> Build a <a href='https://codepen.io' target='_blank'>CodePen.io</a> app that is functionally similar to this: <a href='https://codepen.io/FreeCodeCamp/full/KVNNXY' target='_blank'>https://codepen.io/FreeCodeCamp/full/KVNNXY</a>.",
+        "<strong>Objective:</strong> Build a <a href='https://codepen.io' target='_blank'>CodePen.io</a> app that is functionally similar to this: <a href='https://codepen.io/FreeCodeCamp/pen/xVopBo' target='_blank'>https://codepen.iohttps://codepen.io/FreeCodeCamp/pen/xVopBo</a>.",
```

Comment:
@theflametrooper @DealPete 
It should actually be : `https://codepen.io/FreeCodeCamp/`**full**`/xVopBo`